### PR TITLE
Feat/enhance creating page

### PR DESCRIPTION
**Pull request by Jisay**
ref: MANAGER-2772

### CODE REVIEW

**Comment by antleblanc** [comment]
There is some typo into your commit message:

> feat(pci.projects.project.creating): **display waiting** message

File: packages/manager/modules/pci/src/projects/project/creating/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -12,5 +12,9 @@
   "pci_projects_project_creating_cancel": "Annuler ma commande",
   "pci_projects_project_creating_cancel_title": "{{ orderId }} : annuler ma commande",
   "pci_projects_project_creating_welcome": "Bienvenue chez Public Cloud",
-  "pci_projects_project_creating_almost_done": "Vous y êtes presque…"
+  "pci_projects_project_creating_almost_done": "Vous y êtes presque…",
+  "pci_projects_project_creating_long_time": "Cela peut prendre quelques minutes.",
+  "pci_projects_project_creating_long_time_wait": "En attendant n'hésitez pas à consulter nos guides ou vos autres projets :)",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/creating/translations/Messages_fr_FR.json]: Is `:)` necessary?

File: packages/manager/modules/pci/src/projects/projects.routing.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import template from './projects.html';
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider
     .state('pci.projects', {
-      abstract: true,
+      // abstract: true,
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/projects.routing.js]: Could be removed?

File: packages/manager/modules/pci/src/projects/project/creating/translations/Messages_fr_FR.json (hunk 1 above)
Comment (Jisay) [reply packages/manager/modules/pci/src/projects/project/creating/translations/Messages_fr_FR.json]: Yes :)

File: packages/manager/modules/pci/src/projects/projects.routing.js (hunk 2 above)
Comment (Jisay) [reply packages/manager/modules/pci/src/projects/projects.routing.js]: Yes :)

File: packages/manager/modules/pci/src/projects/project/creating/translations/Messages_fr_FR.json (hunk 1 above)
Comment (antleblanc) [reply packages/manager/modules/pci/src/projects/project/creating/translations/Messages_fr_FR.json]: Ok :)

**Pull request merged**